build-logic/documentation/src/main/groovy/gradlebuild/docs/RenderMarkdown.java

```diff
@@ -39,6 +40,7 @@
 import java.io.InputStreamReader;
 import java.io.OutputStreamWriter;
 import java.nio.charset.Charset;
+import java.util.Arrays;
 import java.util.Collections;
```

Comment:
(after the change below, remove this import)

Reply:
Used arrays instead of collections. Removed collections.